front/components/assistant/conversation/Conversation.tsx

```diff
@@ -86,6 +94,16 @@ export default function Conversation({
 
   const buildEventSourceURL = useCallback(
     (lastEvent: string | null) => {
+      if (!lastConversationTs) {
+        return null;
+      }
+      if (lastConversationTs + 10 * 1000 < Date.now()) {
```

Comment:
This is 10s not 10mn right?

Reply:
Good catch. I was using 10 seconds for testing the behavior. I will fix it depending on the route we decide to take (active window vs last modified time of the conversation. See my comment below).